package/MDAnalysis/coordinates/__init__.py

```diff
@@ -268,6 +272,8 @@ class can choose an appropriate reader automatically.
 
 .. _`netcdf4-python`: https://github.com/Unidata/netcdf4-python
 .. _`H5MD`: https://nongnu.org/h5md/index.html
+.. _`chemfiles`: https://chemfiles.org/
+.. _`list of chemfiles file formats`: https://chemfiles.org/chemfiles/latest/formats.html
```

Comment:
Perhaps we should link to specific versions as you suggest because for reasons that I don't know, we're restricting the chemfiles version
https://github.com/MDAnalysis/mdanalysis/blob/f542aa485983f8d3dd250b36a886061f696c3e97/package/MDAnalysis/coordinates/chemfiles.py#L63-L64
that we can use.

Reply:
Of course, it would be better if we could just say "whatever latest version ≥ 0.9" and then I would leave the link to latest docs.

Reply:
For right now, I link to the 0.9.3 docs.